website/docs/docs/cloud/git/setup-azure-service-principal.md

```diff
@@ -0,0 +1,166 @@
+    - **Directory(tenant) ID:** Found in the Microsoft Entra ID app.
+        <Lightbox src="/img/docs/cloud-integrations/service-principal-fields.png" title="Fields for adding Entra ID app to dbt Cloud."/>
+
+Your Microsoft Entra ID app should now be added to your dbt Cloud Account. People on your team who want to develop in the dbt Cloud IDE or dbt Cloud CLI can now personally [authorize Azure DevOps from their profiles](/docs/cloud/git/authenticate-azure).
+
+
+## Migrate to service principal
+
+Migrate from a service user to service principal using the existing app. It will only take a few steps and you won't experience any service disruptions. 
+
+- Verify whether or not your app has a service principal
+    - If not, create the app service principal
+- Update the application's configuration
+- Update the configuration in dbt Cloud
+
+### Verify the service principal
+
+You will need an Entra ID admin to complete these steps.
+
+To confirm whether your existing app already has a service principal:
+ 
+1. In the Azure account, navigate to **Microsoft Entra ID** -> **Manage** -> **App registrations**.
+2. Click on the application used for the service user integration with dbt Cloud. 
+3. Verify whether there is a name populating the **Managed application in local directory** field. 
+    - If there is a name, continue on from the [add role to your service principal](#add-a-role-to-service-principal) and follow the remaining instructions.
+    - If there is no name, go to the next section, [Create the service principal](#create-the-service-principal).
+
+### Create the service principal
+
+If there is not a name populating that field, this means a Service Principal does not exist. To configure a Service Principal, please review the instructions here.
+
+If your dbt Cloud app does not have a service principal, take the following actions in your Azure account:
+
+1. Navigate to **Microsoft Entra ID**.
+2. Under **Manage** on the left-side menu, click **App registrations**.
+3. Click the app for the dbt Cloud and Azure DevOps integration.
+4. Locate the **Managed application in local directory** field and click **Create Service Principal**.
+
+    <Lightbox src="/img/docs/cloud-integrations/create-service-principal.png" width="80%" title="Example of the 'Create Service Principal' option highlighted ."/>
+
+5. Follow the instructions in [Add permissions to service principal](#add-permissions-to-your-service-principal) if the app doesn't already have them assigned.
+6. In dbt Cloud, navigate to **Account settings** --> **Integrations** and edit the **Azure DevOps** integration.
+7. Click the **Service principal** option, fill out the fields, and click **Save**. The services will continue to function uninterrupted. 
```

Comment:
When customers do the migration they shouldn't need to change anything in the fields if they're using the same Entra ID app, so I'm wondering if we should change wording here.